dockertest/environment.py

```diff
@@ -47,12 +55,7 @@ def get_selinux_context(path):
     When selinux is enabled, return the context of ``path``
     :param path: Full or relative path to a file or directory
     :return: SELinux context as a string or None if not set
+    :raises IOError: When file does not exist.
```

Comment:
>  also wonder if there are other circumstances in which selinux might raise IOError

I'm sure there are.  Hmmm, okay, if you're wondering about this stuff, it's probable someone else will too.  That can all be addressed by docs and comments...